plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/TransactionCachingJdbcClient.java

```diff
@@ -1,91 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.jdbc;
-
-import com.google.common.cache.Cache;
-import com.google.common.cache.CacheBuilder;
-import io.airlift.units.Duration;
-import io.trino.spi.connector.ColumnHandle;
-import io.trino.spi.connector.ConnectorSession;
-import io.trino.spi.predicate.TupleDomain;
-import io.trino.spi.statistics.TableStatistics;
-
-import java.util.Objects;
-import java.util.Set;
-
-import static java.util.Objects.requireNonNull;
-import static java.util.concurrent.TimeUnit.MILLISECONDS;
-
-public class TransactionCachingJdbcClient
-        extends CachingJdbcClient
-{
-    private final Cache<TableStatisticsCacheKey, TableStatistics> statisticsCache;
-
-    public TransactionCachingJdbcClient(JdbcClient delegate, Duration cachingTtl)
-    {
-        // session stays the same per transaction, therefore session properties don't need to
-        // be a part of cache keys in CachingJdbcClient
```

Comment:
Shouldn't this comment be addressed?
Seems stats is cached among different sessions now.
If yes, please add test for this.